src/main/scala/com/typesafe/sbt/site/SitePreviewPlugin.scala

```diff
@@ -34,22 +36,25 @@ object SitePreviewPlugin extends AutoPlugin {
           waitForKey()
       }
       if(browser)
-        Browser open ("http://localhost:%d/" format port)
+        Browser open ("http://localhost:%d/%s".format(port, path))
       waitForKey()
       server stop()
       server destroy()
     },
     previewAuto := {
       val port = previewFixedPort.value getOrElse Port.any
       val browser = previewLaunchBrowser.value
+      val path = previewPath.value
 
       Preview(port, (target in previewAuto).value, makeSite, Compat.genSources, state.value) run { server =>
         if (browser)
           Browser open(server.portBindings.head.url)
-      }
+          Browser open(server.portBindings.head.url + "/" + path)
```

Comment:
Shouldn't this line modify the previous one?

Reply:
Thanks for noticing, I guess I messed up with `master` vs `develop`.